#Diplay image or not
Display: false

#Choose detection feature method, can only use one at a time
Yolo_Center_ROI: true
Blob_tracking: false

#Parameter for the control law related to the arm use
error_max: 0.05 #error quadratic max to stop the loop, depend on the precision you need.
end_effector_velocities_max: 10 #set the velocities max of the end effector in the camera frame.
depth_desired: 0.16 # depth that is desired to reach
command_gripper: 0.25 # command to send to close the gripper

#Adaptative gain value, depending on the sended velocities and low pass filter gain
gain_for_low_value: 15
gain_for_high_value: 0.8
gain_slope: 30
gain_low_pass_filter: 0.5

#You can plot real time velocities curves, for debug or performance comparation
plot: false #set true of you want to plot real time curves of the speed and position
save: false #set true if you want to save curves
